### PR TITLE
`[MirrorExtension]` SetIntercom and ResetIntercom

### PR DESCRIPTION
**Pull request by NotZer0Two**
This PR adds the possibility to a Developer via Resync and Sync to change the text for a specific player

```
//Set Display
player.SetIntercomDisplayTextForTargetOnly("Test");

//Reset
MirrorExtension.ResetIntercomDisplayTextForTargetOnly();
```

### CODE REVIEW

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 1]
```diff
@@ -25,6 +25,8 @@ namespace Exiled.API.Extensions
     using PlayerRoles;
     using PlayerRoles.FirstPersonControl;
     using PlayerRoles.PlayableScps.Scp049.Zombies;
+    using PlayerRoles.Voice;
+    using PluginAPI.Core;
```
Comment (xNexusACS) [inline Exiled.API/Extensions/MirrorExtensions.cs:29]: PluginAPI.Core is not needed here

**Review by xNexusACS — CHANGES_REQUESTED** [review]

**Comment by xNexusACS** [comment]
fix conflicts

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 2]
```diff
@@ -193,6 +194,24 @@ public static void SetRoomColorForTargetOnly(this Room room, Player target, Colo
             target.SendFakeSyncVar(room.RoomLightControllerNetIdentity, typeof(RoomLightController), nameof(RoomLightController.NetworkOverrideColor), true);
         }
 
+        /// <summary>
+        /// Sets <see cref="Intercom.DisplayText"/> that only the <paramref name="target"/> player can see.
+        /// </summary>
+        /// <param name="target">Only this player can see Display Text.</param>
+        /// <param name="text">Text displayed to the player</param>
+        public static void SetIntercomDisplayTextForTargetOnly(this Player target, string text)
```
Comment (ITeMbI4) [inline Exiled.API/Extensions/MirrorExtensions.cs:202]: It can be done via =>

File: Exiled.API/Extensions/MirrorExtensions.cs [hunk 3]
```diff
@@ -193,6 +194,24 @@ public static void SetRoomColorForTargetOnly(this Room room, Player target, Colo
...
+        {
+            target.SendFakeSyncVar(IntercomDisplay._singleton.netIdentity, typeof(IntercomDisplay), nameof(IntercomDisplay.Network_overrideText), text);
+        }
+
+        /// <summary>
+        /// Resync <see cref="Intercom.DisplayText"/>
+        /// </summary>
+        public static void ResetIntercomDisplayText()
+        {
+            ResyncSyncVar(IntercomDisplay._singleton.netIdentity, typeof(IntercomDisplay), nameof(IntercomDisplay.Network_overrideText));
```
Comment (ITeMbI4) [inline Exiled.API/Extensions/MirrorExtensions.cs:212]: It can be done via =>

Comment (NotZer0Two) [reply Exiled.API/Extensions/MirrorExtensions.cs]: Done

**Review by xNexusACS — APPROVED** [review]

**Pull request merged**